.github/CONTRIBUTING.md

```diff
@@ -11,7 +11,7 @@ is a great boon to your development process.
 To get ready to work on the codebase, please do the following:
 
 1. Fork & clone the repository, and make sure you're on the **main** branch
-2. Run `npm ci`
+2. Run `yarn` ([install](https://yarnpkg.com/getting-started/install))
```

Comment:
The flag keeps the lockfile unchanged and installs everything with the locked versions, similar to `npm ci`, and this also makes the install step *a lot faster*:
```suggestion
2. Run `yarn --immutable` ([install](https://yarnpkg.com/getting-started/install))
```

This is a Yarn v3 flag, which we use since 37c1cb4.